### PR TITLE
frontend: autopilot: Add information about each firmware version type

### LINKED ISSUE #938: Firmware update page should have short descriptions of what each firmware means
Official - stable release bla bla bla whiskas sachê
Beta - some additional upgrades, soon to be stable
Dev - Development branch, all newest features, unstable and possibly dangerious

### PR DESCRIPTION
**Pull request by rafaellehmkuhl**
Fix #938 

@patrickelectric @Williangalvani @ES-Alexander @joaoantoniocardoso please check the texts to see if they match what you think should be explained about each version.

### CODE REVIEW

**Review by Williangalvani — APPROVED** [review]

File: core/frontend/src/components/autopilot/FirmwareManager.vue [hunk 1]
```diff
@@ -65,17 +65,35 @@
...
+              class="ml-4"
+              v-bind="attrs"
+              v-on="on"
+            >
+              mdi-information
+            </v-icon>
+          </template>
+          <p>Stable - Production-ready release, meant to be used by most users.</p>
+          <p>Official - Latest stable release available.</p>
+          <p>Beta - In-tests release, adding new features, aiming to become a stable release. May have bugs.</p>
```
Comment (ES-Alexander) [inline core/frontend/src/components/autopilot/FirmwareManager.vue:93]:
```suggestion
          <p>Beta - In-testing release with new features and improvements; aiming to become stable. May have bugs.</p>
```

File: core/frontend/src/components/autopilot/FirmwareManager.vue [hunk 2]
```diff
@@ -65,17 +65,35 @@
...
+          <p>Dev - Development branch, with all newest features, but probably unstable and possibly dangerous.</p>
```
Comment (ES-Alexander) [inline core/frontend/src/components/autopilot/FirmwareManager.vue:94]:
```suggestion
          <p>Dev - Development branch, with all the newest features. Intentionally unstable (changes quickly), and possibly untested/dangerous.</p>
```

File: core/frontend/src/components/autopilot/FirmwareManager.vue [hunk 3]
```diff
@@ -65,17 +65,35 @@
...
+          <template #activator="{ on, attrs }">
+            <v-icon
+              class="ml-4"
+              v-bind="attrs"
+              v-on="on"
+            >
+              mdi-information
+            </v-icon>
+          </template>
+          <p>Stable - Production-ready release, meant to be used by most users.</p>
```
Comment (ES-Alexander) [inline core/frontend/src/components/autopilot/FirmwareManager.vue:91]:
```suggestion
          <p>Stable - A production-ready release. Suitable for most users.</p>
```

File: core/frontend/src/components/autopilot/FirmwareManager.vue [hunk 4]
```diff
@@ -65,17 +65,35 @@
...
+          <p>Official - Latest stable release available.</p>
```
Comment (ES-Alexander) [inline core/frontend/src/components/autopilot/FirmwareManager.vue:92]:
```suggestion
          <p>Official - The latest stable release. Recommended for most users.</p>
```

**Review by ES-Alexander — COMMENTED** [review]
Good to have some clarity on this - thanks! :-)

Can’t remember what the current results ordering is (and that might be unrelated to this), but I think it would be good if we could have `official, beta, dev, [past stables…]` in the dropdown. I think that strikes a good mix of encouraging stability while discouraging old/outdated versions, but still providing them for people that need/really want them.

**Review by patrickelectric — APPROVED** [review]

**Comment by ES-Alexander** [comment]
> > Good to have some clarity on this - thanks! :-)
> > Can’t remember what the current results ordering is (and that might be unrelated to this), but I think it would be good if we could have `official, beta, dev, [past stables…]` in the dropdown. I think that strikes a good mix of encouraging stability while discouraging old/outdated versions, but still providing them for people that need/really want them.
> 
> Everything implemented! Thanks for the suggestions eliot!
> 
> image

Nice! 
Is it possible to change the order of stable to be latest first? Users most likely want the most recent stable over the oldest saved one.

**Pull request merged**